### DISCUSSION TITLE
Color Error: RGB is BGR

### DISCUSSION THREAD

**Discussion by LFernandoMB** [rust-skia/rust-skia#937] [General]
Dear all, how are you?

I have a curious error, I have a demo using Skia and Vulkan and I create an image with a circle for example, and everything works correctly.
However, I have a demo using Skia and the CPU and the same commands to generate a circle result in the inversion of the RGB colors between red and blue, so when I try to draw a red circle it draws blue and vice versa.

My function:

```rust
fn draw_circle(canvas: &mut skia_safe::Canvas) {
        let screen_center = SCREEN_CENTER;
        let mut circle_paint =
            skia_safe::Paint::new(skia_safe::Color4f::new(1.0, 0.0, 0.0, 1.0), None);
        circle_paint.set_anti_alias(true);
        circle_paint.set_style(skia_safe::paint::Style::Fill);

        let circle_center = skia_safe::Point::new(screen_center.0, screen_center.1);
        let circle_radius = 200.0;
        canvas.draw_circle(circle_center, circle_radius, &circle_paint);
    }
``` 

Do you know if there is a specific reason for this discrepancy, or if it is an error in my code?

**Comment by pragmatrix** [comment]
Perhaps related to #636 and/or #802.
